CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 -
 
+# v2.18.0
+
+- [added] Added support for specifying the analytics label for notifications.
```

Comment:
Thought you got rid of it :)

Reply:
Still keeping it for the moment. Possibly after this release :)